instrumentation/opentelemetry-instrumentation-boto3sqs/src/opentelemetry/instrumentation/boto3sqs/__init__.py

```diff
@@ -0,0 +1,428 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Instrument `boto3sqs`_ to trace SQS applications.
+
+.. _boto3sqs: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html
+
+
+Usage
+-----
+
+.. code:: python
+
+    import boto3
+    from opentelemetry.instrumentation.boto3sqs import Boto3SQSInstrumentor
+
+
+    Boto3SQSInstrumentor().instrument()
+"""
+import logging
+from typing import Any, Collection, Dict, Generator, List, Optional
+
+import boto3
+import botocore.client
+import wrapt
+
+from opentelemetry import context, propagate, trace
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import (
+    _SUPPRESS_INSTRUMENTATION_KEY,
+    unwrap,
+)
+from opentelemetry.propagators.textmap import CarrierT, Getter, Setter
+from opentelemetry.semconv.trace import (
+    MessagingDestinationKindValues,
+    MessagingOperationValues,
+    SpanAttributes,
+)
+from opentelemetry.trace import Link, Span, SpanKind, Tracer, TracerProvider
+
+from .package import _instruments
+from .version import __version__
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
_logger = logging.getLogger(__name__)
```
Everything should be private unless it is intended to be exposed to the user.

Reply:
Done